modules/geo-layers/src/mvt-layer/coordinate-transform.js

```diff
@@ -0,0 +1,52 @@
+const availableTransformations = {
+  Point,
+  MultiPoint,
+  LineString,
+  MultiLineString,
+  Polygon,
+  MultiPolygon
+};
+
+function Point([pointX, pointY], {x, y, z}) {
+  const originX = x;
+  const originY = y;
+  const zoomLevelSize = Math.pow(2, z);
+
+  const y2 = 180 - ((pointY + originY) * 360) / zoomLevelSize;
+
+  return [
+    ((pointX + originX) * 360) / zoomLevelSize - 180,
+    (360 / Math.PI) * Math.atan(Math.exp((y2 * Math.PI) / 180)) - 90
+  ];
+}
+
+function getPoints(geometry, tile) {
+  return geometry.map(g => availableTransformations.Point(g, tile));
```

Comment:
Any reason to reference the wrapper object instead of calling the function directly?

Reply:
It works as an enum, also to dynamically call the functions

Reply:
But you are right, there is not necessary :+1: